src/generator.ts

```diff
@@ -47,7 +47,11 @@ function generateExpression(expr:Expression) {
   switch (expr.type) {
     case 'whitespace':
     case 'code': return expr.value;
-    case 'text': return `Html.text "${expr.value.replace(/"/g, '\\"')}"`;
+    case 'text': {
+      let multiline = expr.value.match(/\n|\r/) ? true : false;
+      let quote = multiline ? '"""' : '"';
+      return `Html.text ${quote}${expr.value.replace(/"/g, '\\"')}${quote}`;
```

Comment:
Also, here....
```ts
const escapedQuote = quote.replace(/"/g, '\\"');
return Html.text ${quote}${expr.value.replace(new RegExp(quote, 'g'), escapedQuote)}${quote};
```

:point_up: see if this fixes the failing test (and allows you to drop the `\`s in the cucumber steps (since the actual output we should produce should be `Html.text """` and not `Html.text \"\"\"`).